packages/vue-components/package.json

```diff
@@ -41,6 +41,7 @@
     "jest": "^27.5.1",
     "jest-serializer-vue": "^2.0.2",
     "portal-vue": "^2.1.7",
+    "vue-final-modal": "2.3.2",
```

Comment:
Just checking on this, I noticed that their latest version for Vue 2 is [`2.4.1`](https://www.npmjs.com/package/vue-final-modal). Is there a reason for using this specific version? 

Asking this because I saw in their [`2.4.0` release](https://github.com/vue-final/vue-final-modal/releases/tag/v2.4.0), they added the feature to import component directly. For example,
```js
import { vfmPlugin, $vfm, VueFinalModal } from 'vue-final-modal'
``` 
This should theoretically reduce the size of the bundle as only the required components are imported.

Reply:
Vue Final Modal (VFM) v2.4.1 requires Vue v2.6.14 as a dependency. However, upgrading Vue from v2.6.11 (current) to v2.6.14 causes hydration errors in Navbars. The hydration errors should be unrelated to this issue - they are present even when upgrading on the master branch.
The latest version of VFM compatible with Vue v2.6.11 is VFM v2.3.2.

Reply:
> The latest version of VFM compatible with Vue v2.6.11 is VFM v2.3.2.

Ahh ok got it :)

> However, upgrading Vue from v2.6.11 (current) to v2.6.14 causes hydration errors in Navbars. The hydration errors should be unrelated to this issue - they are present even when upgrading on the master branch.

Have you tried updating the dev version specified in [Site/index.js](https://github.com/MarkBind/markbind/blob/master/packages/core/src/Site/index.js#L319) to v2.6.14 as well to see if the hydration issue is still present? Might be cause by a mismatch of Vue's version. Will also need to update the [vue.min.js](https://github.com/MarkBind/markbind/blob/master/packages/core-web/asset/js/vue.min.js), which is used in production.